requirements/docs.txt

```diff
@@ -0,0 +1,8 @@
+pytest
+coverage
+matplotlib
+sphinx==1.6.2
+pillow
+sphinx-gallery
+numpydoc
+# Note that CirclecI uses dev numpydoc (for now)
```

Comment:
building the docs requires running the examples which requires pandas and scikit-image, right?